src/Player.h

```diff
@@ -20,7 +20,9 @@ class Player : public Ship {
 	Player(const Json &jsonObj, Space *space);
 	Player(const ShipType::Id &shipId);
 
-	virtual void SetDockedWith(SpaceStation *, int port) override;
+	void OnDocked(SpaceStation *, int) override;
+	void OnUndocked(SpaceStation *, int) override;
+
```

Comment:
Curious on why `SetDockedWith` is split up?

Reply:
It's not split up; this just removes an unneeded override of the base class. `OnDocked`/`OnUndocked` are event handlers called well after `SetDockedWith` has already returned.